src/ConfigAggregator.php

```diff
@@ -92,6 +96,35 @@ public function getMergedConfig()
         return $this->config;
     }
 
+    /**
+     * Throw an exception if duplicate config providers are found.
+     *
+     * @param ProviderIterable $providers
+     * @return ProviderIterable
+     * @throws InvalidConfigProviderException
+     */
+    private function validateNoDuplicateProviders(iterable $providers): iterable
```

Comment:
I'd prefer this method to be named `assertSomething` rather than `validateSomething` because it is an assertion that will throw an exception as opposed to a validation that would typically return success or failure - Just a nitpick and not a blocker for me.

Reply:
`validateNoDuplicateProviders` -> `assertNoDuplicateProviders`?

Reply:
I'm not _that_ picky, but yeah, something like that, or `assertUniqueConfigProviders`?

As an assertion, you could drop the return value entirely, i.e. `return void`

Reply:
Btw, the current method naming comes from [this conversation](https://github.com/laminas/laminas-config-aggregator/pull/47/commits/5182755b9bc7e747bcc714560e025ea7f8919af6#r1709310928) with @samsonasik and later [this conversation](https://github.com/laminas/laminas-config-aggregator/pull/47/commits/ccec0e4c82f6f896d052726a6ec2120b31ded642#r1709519256) with @Xerkus

Reply:
😂 - No worries! Like I said - it's not a blocker for me 👍